library/src/amd_detail/rocblaslt/src/UserDrivenTuningParser.cpp

```diff
@@ -0,0 +1,166 @@
+#include "UserDrivenTuningParser.hpp"
+
+#include <fstream>
+#include <sstream>
+#include <utility>
+
+namespace TensileLite
+{
+
+    std::multimap<ProblemOverride, int> getContractionProblemsFromFile(const std::string& path)
+    {
+
+        static std::multimap<ProblemOverride, int> m_override;
```

Comment:
Do you have protection for m_override for multi-thread cases?

Reply:
Thanks, it’s done in commit [ff3ab5] (https://github.com/ROCm/hipBLASLt/pull/1289/commits/ff3ab5fc397d4a07745b3955e3a750bdaf866624)